src/main/java/org/jabref/gui/bibtexkeypattern/ResolveDuplicateLabelDialog.java

```diff
@@ -59,7 +58,7 @@ public ResolveDuplicateLabelDialog(BasePanel panel, String key, List<BibEntry> e
             JCheckBox cb = new JCheckBox(Localization.lang("Generate BibTeX key"), !first);
             b.appendRows("1dlu, p");
             b.add(cb).xy(1, row);
-            PreviewPanel previewPanel = new PreviewPanel(null, null, Globals.getKeyPrefs(), Globals.prefs.getPreviewPreferences(), new FXDialogService(), ExternalFileTypes.getInstance());
```

Comment:
Can you please convert this dialog also to JavaFX (in a follow-up PR). Thanks

Reply:
yep  on my list. Added this also to #3861